build/gradle/commitAndProductionStage.gradle

```diff
@@ -826,6 +844,34 @@ def packageServer(productOutputPath) {
     }
 }
 
+/**
+ * Method executed only for OS X. 
+ * To run the HALE studio in mac, JRE must be bundled within the HALE studio app
+ * and its location must be added in the HALE.ini file.
+ * This method is a work around to modify HALE.ini file.
+ * This method reads HALE.ini file stored under "~/Eclipse.app/Contents/Eclipse/HALE.ini",
+ * then appends "-vm" arguments along with the JRE location bundled within the application.
+ * Finally, the changes are written back to the "HALE.ini" file.
+ * filepath HALE.ini file path
+ */
+def updateHaleIniFileWithJrePath(filepath) {
+
+	File file = new File(filepath)
+	def fileContent = file.getText();
+
+	def indexOfVmargs = fileContent.indexOf("-vmargs")
+	
+	def updatedContent = fileContent.substring(0, indexOfVmargs)
+	.concat("-vm")
+	.concat("\n")
+	.concat("../Eclipse/jre/jdk8u282-b08/Contents/Home/jre/bin/java")
```

Comment:
Can't we get rid of the `jdk8u282-b08` here? This would mean that when the bundled JRE is changed this code needs to be changed as well. It would be better of only the reference to the JRE artifact would need to be changed,

Reply:
I have done the change to strip `jdk8u282-b08` during `tar.gz` extraction. As for all the Mac jdk bundles, root folder is the name of the jdk, so I'm stripping the root folder name and directly copying the `Contents` into `jre` folder, thereby getting rid of `jdk8u282-b08`.